apps/roam/src/components/settings/DiscourseNodeConfigPanel.tsx

```diff
@@ -32,6 +35,12 @@ const DiscourseNodeConfigPanel: React.FC<DiscourseNodeConfigPanelProps> = ({
     null,
   );
 
+  const [isAlertOpen, setIsAlertOpen] = useState(false);
+  const [alertMessage, setAlertMessage] = useState("");
+  const [alertConfirmAction, setAlertConfirmAction] = useState<
+    () => Promise<void>
+  >(() => Promise.resolve());
```

Comment:
This state is unnecessary.  It can be a function or happen completely in the `onConfirm`